src/backend/src/services/reimbursement-requests.services.ts

```diff
@@ -880,4 +880,30 @@ export default class ReimbursementRequestService {
 
     return vendorTransformer(vendor);
   }
+
+  /**
+   * Deletes the vendor
+   *
+   * @param vendorId the requested vendor to be deleted
+   * @param submitter the user deleting the vendor
+   * @returns the 'deleted' vendor
+   */
+  static async deleteVendor(vendorId: string, submitter: User) {
```

Comment:
in this service method, u still have 'Only admins or someone on the finance team should be able to delete a vendor' to implement. there's examples of this already in other endpoints.